frontend/Lighting.jsx

```diff
@@ -67,43 +69,78 @@ function Lighting() {
   }), [lightOn, lightColor, lightIntensity]));
 
   return (
-    <View style={styles.container}>
-      <Text>
-        {'Turn Light On\n\n'}
-      </Text>
-      <Switch
-        trackColor={{ false: '00FF00', true: 'FFFFFF' }}
-        onValueChange={(value) => setLightOn(value)}
-        value={lightOn}
-        style={{ transform: [{ scaleX: 2 }, { scaleY: 2 }] }}
-      />
-      <Text>
-        {`\n\n\n\nIntensity of Light: ${Math.round(lightIntensity)}`}
-      </Text>
-      <Slider
-        minimumValue={0}
-        maximumValue={3}
-        disabled={!lightOn}
-        value={lightIntensity}
-        onValueChange={(value) => setLightIntensity(value)}
-        style={{ width: 160, height: 50 }}
-        minimumTrackTintColor="#ffd700"
-        maximumTrackTintColor="#fafad2"
-      />
-      <Text>
+    <LinearGradient
+      colors={['#F9F6FF', '#CFDFF7']}
+      style={{
+        flex: 1,
+        padding: 10,
+        paddingTop: 80,
+        alignItems: 'center',
+      }}
+    >
+      <View style={{ flexDirection: 'row', justifyContent: 'space-between', marginBottom: 30 }}>
```

Comment:
It looks like there could probably use some padding or margin at the top as the LightingTitle and switch are very close to the header

<img width="478" alt="image" src="https://user-images.githubusercontent.com/3270324/106522732-f7303080-6494-11eb-9bb2-e353f8af326d.png">